.github/workflows/build_pull_request.yml

```diff
@@ -1,6 +1,8 @@
 name: build_pull_request
 
 on:
+  branches:
+    - 'hacl-star-*'
```

Comment:
The docs all use `**` so maybe we should too?

Reply:
I tested the `*` syntax on another repo and it works